storage/ndb/src/kernel/vm/SimulatedBlock.hpp

```diff
@@ -583,8 +583,7 @@ class alignas(NDB_CL) SimulatedBlock :
   {
     return globalData.ndbMtLqhThreads +
            globalData.ndbMtTcThreads +
-           globalData.ndbMtMainThreads +
-           globalData.ndbMtRecoverThreads;
+           globalData.ndbMtMainThreads;
```

Comment:
main, rep threads reside behind recv threads, it seems that we shouldn't add ndbMtMainThreads here.

Reply:
Good catch.